lib/astutils.h

```diff
@@ -71,8 +72,17 @@ bool isWithoutSideEffects(bool cpp, const Token* tok);
 /** Is scope a return scope (scope will unconditionally return) */
 bool isReturnScope(const Token *endToken);
 
+/** Is varibale changed by function call?
+ * In case the anser of the question is inconcluse, e.g. because the function declaration is not known
+ * the return value is false and the output parameter inconclusive is set to true
+ *
+ * @param tok           token of varible in function call
+ * @param inconclusive  output variable which indicates that the answer of the question is inconclusive
+ */
+bool isVariableChangedByFunctionCall(const Token *tok, const Settings *settings, bool &inconclusive);
```

Comment:
I would like that inconclusive is a pointer to make it more visible in the function calls that the address is passed to the function and that it's an output parameter.